products/views.py

```diff
@@ -45,3 +46,19 @@ class ProductColourListView(ListAPIView):
 class ProductSizeListView(ListAPIView):
     queryset = ProductSize.objects.all()
     serializer_class = ProductSizeListSerializer
+
+
+class ProductReviewsListView(ListAPIView):
+    queryset = ProductReview.objects.all()
+    serializer_class = ProductReviewListSerializer
+
+    def get_queryset(self):
+        product_id = self.request.query_params.get('product_id')
+        if product_id is None:
+            return self.queryset.filter(product_id=product_id)
```

Comment:
product_id None bo'lsa qanday qilib filter qilasiz ?

is not None deb tekshirsangiz to'g'ri bo'lardi

Reply:
O'zi is not None qilmoqchdidim qob ketibdi uzr